rotkehlchen/chain/ethereum/interfaces/balances.py

```diff
@@ -53,6 +53,7 @@
     'uniswap-v3',
     'hedgey',
     'hyperliquid',
+    'pendle',
```

Comment:
is it only in Ethereum?

Reply:
> is it only in Ethereum?

actions can only be performed on ethereum. the state is then synced with other evm chains.